tests/integration.rs

```diff
@@ -60,6 +60,7 @@ fn integration_test() {
             "warn",
             "-Wclippy::pedantic",
             "-Wclippy::nursery",
+            "--message-format=short",
```

Comment:
I think it needs to come before the `--`

Reply:
You're right. Fixing that immediately.